### PR TITLE
Docs key concepts

### LINKED ISSUE #6: Documentation of key concepts
Use a gdoc first and when all validate it - to ensure sligned understanding - add into https://tstore.readthedocs.io/en/latest/00_introduction.html

### LINKED ISSUE #20: Tutorials with first use cases
- [ ] Tuto 1 : transformations between formats
- [ ] Tuto 2 : Read/write filtering spatially (geoParquet)
- [ ] Tuto 3 : Read/write filtering temporally

### PR DESCRIPTION
**Pull request by sphamba**
# Prework

- [ ] I understand and agree to this repository's [code of conduct](https://github.com/ltelab/tstore/blob/main/CODE_OF_CONDUCT.md).
- [ ] I understand and agree to this repository's [contributing guidelines](https://github.com/ltelab/tstore/blob/main/CONTRIBUTING.md).
- [ ] I have already submitted an [issue](https://github.com/ltelab/tstore/issues) or [discussion thread](https://github.com/ltelab/tstore/discussions) to discuss my idea with the maintainers.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Tutorial
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and communicate accordingly:

**The PR fulfills these requirements:**

- [ ] It's submitted to the branch named as follow:
  - Fix a bug: `bugfix-<some_key>-<word>`
  - Improve the doc: `doc-<some_key>-<word>`
  - Improve a tutorial `tutorial-<some_key>-<word>`
  - Add a new feature: `feature-<some_key>-<word>`
  - Refactor some code: `refactor-<some_key>-<word>`
  - Optimize some code: `optimize-<some_key>-<word>`
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] Don't forget to link PR to issue if you are solving one.
- [ ] All tests are passing.
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

# Related GitHub issues and pull requests

- closes #6
- closes #20

# Summary

_Please explain the purpose and scope of your contribution._

### CODE REVIEW

File: docs/source/01_key_concepts.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Additionally, it may be appropriate to store the observations of different variables in separate `TSArray` objects, which can be combined in a `TSDF` object. This is particularly useful when a station is equipped with multiple sensors to record different variables, which may have different observation frequencies. In fact, variables such as precipitation may have long periods without records, which would result in many NaN values in a multivariate time series.
+
+------
+TSLong
+------
+
+In a **long-form data table**, each variable corresponds to a column and each observation corresponds to a row.
```
Comment (ghiggi) [inline docs/source/01_key_concepts.rst:35]: I would expand this: 
- duplicated timestamps (when simultaneous measurements at multiple stations) 
- lot of NaN if irregular data 
- how to deal with temporal resolutions  
- ...

File: docs/source/01_key_concepts.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+------
+TSWide
+------
+
+In a **wide-form data table**, columns and rows contain levels of a variable. Each observation is defined by its value at the cell and its coordinates in both the row and column indices.
```
Comment (ghiggi) [inline docs/source/01_key_concepts.rst:41]: Makes not many sense to me this description. 
Point to say: 
- data format to represent the values of a univariate timeseries recorded across multiple stations 
- row and columns corresponds to timestamp and station index. 
- Cell values corresponds to the timeseries value at given timestamp for a given station. 
- Format typically used for statistical analysis (i.e. PCA) 
- Requires regular timeseries (same timestamps) to be useful

File: docs/source/index.rst [hunk 3]
```diff
@@ -6,15 +6,15 @@
...
 
-With our software, you can:
+With **TStore**, you can:
 
-1. **TODO**: Quickly search and download the data you need.
+1. Efficiently **store and read** *(probably almost) any form* time series data to disk. Thanks to its hierarchically-structured approach based on `Apache Parquet <https://parquet.apache.org>`__ and `GeoParquet <https://github.com/opengeospatial/geoparquet>`__, you can choose how samples and variables are stored and temporally partitioned.
 
-2. **TODO**: Find the downloaded files on your local storage without digging through folders.
+2. Process complex time series data structures thanks to the `TSDF` object, which by encapsulating univariate or multivariate time series data into an `Apache Arrow structure <https://arrow.apache.org/docs/python/data.html>`__ `TS` (which can be *zero-copy transformed* into pandas and polars objects), allows you to organize heterogeneous time series across samples and variables using a data frame object.
```
Comment (ghiggi) [inline docs/source/index.rst:15]: This sentence make no sense. Something is missing ...

File: docs/source/index.rst [hunk 4]
```diff
@@ -6,15 +6,15 @@
...
 
-The **TStore** is a Python package designed to make your life easier, whether you aim TODO.
-Our goal is to empower you to focus more on what you can discover and create with the data, rather than getting bogged down by the process of handling it.
+The **TStore** is a Python package designed to make your life easier when dealing with time series data. Our goal is to empower you to focus more on what you can discover and create with the data, rather than getting bogged down by the process of handling it.
 
-With our software, you can:
+With **TStore**, you can:
 
-1. **TODO**: Quickly search and download the data you need.
+1. Efficiently **store and read** *(probably almost) any form* time series data to disk. Thanks to its hierarchically-structured approach based on `Apache Parquet <https://parquet.apache.org>`__ and `GeoParquet <https://github.com/opengeospatial/geoparquet>`__, you can choose how samples and variables are stored and temporally partitioned.
```
Comment (ghiggi) [inline docs/source/index.rst:14]: Read TSTORE data in the table format which is most suited to your analysis 
Multiple dataframe backend supported: pandas, dask, polars, pyarrow 
Enable distributed, lazy and and parallel processing 
Seamless integration with geopandas and xvec to manipulate spatial temporal database
Add something on irregular time series ... 
.....

File: docs/source/index.rst [hunk 5]
```diff
@@ -26,20 +26,18 @@ It's a great place to connect with others and get support.
 
 Let's get started and unlock the full potential of the GPM data archive together!
```
Comment (ghiggi) [inline docs/source/index.rst:27]:
```suggestion
Let's get started and unlock the full potential of TSTORE archiving together!

```

File: README.md [hunk 6]
```diff
@@ -19,9 +19,12 @@
 
 ## 🚀 Quick start
 
-Flexible storage for time series
+tstore is a library for flexible storage and processing of time series data.
 
-TODO
+#### ✨ Features
+
+- **efficient storage**: hierarchically-structured specification to efficiently store geospatial time series data based on [Apache Parquet](https://parquet.apache.org) and [GeoParquet](https://github.com/opengeospatial/geoparquet).
```
[the remark this answers is not in the document]
Comment (martibosch) [reply README.md:26]: it is a tiny detail but I have writing "TStore" everywhere, should I change it to "TSTORE" instead?

File: tutorials/01-your-first-tstore.ipynb
Comment (sphamba) [inline tutorials/01-your-first-tstore.ipynb:1]: To run, the following packages are needed (and not specified clearly anywhere): `contextily osmnx tqdm`

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ghiggi) [reply README.md:26]: **TStore**

**Pull request closed without merging**